xen/arch/arm/coproc/coproc.h

```diff
@@ -47,25 +49,86 @@ struct vcoproc_info {
     /* list is used to append instances of vcoproc to vcoprocs_list */
     struct list_head list;
 
+    /* TODO */
+    bool_t is_running;
```

Comment:
I would introduce a state with macro at the moment, 'cause we likely would have number of states